vector/src/main/java/im/vector/app/features/notifications/NotificationDrawerManager.kt

```diff
@@ -457,7 +457,7 @@ class NotificationDrawerManager @Inject constructor(private val context: Context
 
             if (eventList.isEmpty() || eventList.all { it.isRedacted }) {
                 notificationUtils.cancelNotificationMessage(null, SUMMARY_NOTIFICATION_ID)
-            } else {
+            } else if (hasNewEvent) {
```

Comment:
can confirm this fixes the issue, great catch! 

it might be worth renaming the variable to `hasGroupChanged` as what this logic is doing is updating the summary group notification when one of the group children has also been notified 

I'm also happy to make the change in a separate PR as I'll be in the area attempting to address the double sound issue

Reply:
thanks a lot Adam, I think we can merge the PR now!